numba_cuda/numba/cuda/cudadrv/libs.py

```diff
@@ -173,4 +191,14 @@ def test():
         print('\tERROR: failed to find %s:\n%s' % (lib, e))
         failed = True
 
+    include = get_cuda_include_dir()
+    # Check cuda include paths
+    try:
+        print('\tChecking include directory', end='...')
```

Comment:
This causes the end of the output of `libs.test()` to be:

```
Finding libdevice from Conda environment (NVIDIA package)
	Located at /home/gmarkall/miniforge3/envs/numbadev/nvvm/libdevice/libdevice.10.bc
	Checking library...	ok
	Checking include directory...	ok
```

It's missing a part that should be something like:

```
Finding include directory from <method>
        Located at <location>
```

so that the output would look like at the end:

```
Finding libdevice from Conda environment (NVIDIA package)
	Located at /home/gmarkall/miniforge3/envs/numbadev/nvvm/libdevice/libdevice.10.bc
	Checking library...	ok
Finding include directory from Conda environment
        Located at /home/gmarkall/miniforge3/envs/numbadev/targets/x86_64-linux/include
	Checking include directory...	ok
```

Reply:
It might also be handy to print the value of `CUDA_INCLUDE_PATH` here, so that all the information is available to the user (even if we don't test for anything in that location)

Reply:
Fixed in https://github.com/NVIDIA/numba-cuda/pull/17/commits/6612e0d49c51e22a1b884fd97b6cc7f51e49c2ae